docker/Makefile

```diff
@@ -37,6 +37,13 @@ endif
 build:
 	docker-compose build mail app ld-api
 
+.PHONY: scan
+# https://docs.docker.com/engine/scan
+scan:
+	docker build -t lf-app:prod -f app/Dockerfile --platform linux/amd64 ..
+	docker login
+	-docker scan --accept-license lf-app:prod > docker-scan-results.txt
```

Comment:
Is there a stray dash here?  `-docker scan`
I don't understand this syntax

Reply:
OK I learned something: https://stackoverflow.com/questions/3477292/what-do-and-do-as-prefixes-to-recipe-lines-in-make
The `-` preceding a command ignores the exit status of the command so that `make` will complete successfully without error.

Reply:
yes, if the scan finds an error it will return a `1` probably for CI environments...we don't really care about that when redirecting the output to a file since we'll be reviewing the file for vuln's anyway.